removed swapcomplete messages from protocol (#18)
opendexnetwork/opendex: 04-swap-protocol.md

3. Taker sends the `SwapRequest` message to the maker, which includes `r_hash`
4. Maker confirms full or partial quantity in the `SwapAccepted` message
5. Taker starts the swap by dispatching the first-leg HTLCs on the DAI payment channel to the maker end, using `r_hash`
6. Maker listens for an incoming HTLC on the DAI payment channel. Once it arrives he verifies price and quantity and then dispatches the second-leg HTLCs on the BTC payment channel to the taker end.
7. Taker listens for an incoming HTLC on the BTC payment channel. Once it arrives he releases `r_preimage`. This allows **both** the taker and the maker payments to finalize.

> 9. Both nodes locally mark the swap as completed once the respective HTLC is resolved and the payment is finalized.

Reviewer: ```suggestion
8. Both nodes locally mark the swap as completed once the respective HTLC is resolved and the payment is finalized.
```

Reviewer: GitBook corrects this automatically, but for someone viewing the plain markdown files we want numbering to be correct.

5. Taker starts the swap by dispatching the first-leg HTLCs on the DAI payment channel to the maker end, using `r_hash`
6. Maker is listening for an incoming HTLC on the DAI payment channel. Once it arrives he verifies price and quantity and then dispatches the second-leg HTLCs on the BTC payment channel to the taker end.
7. Taker is listening for an incoming HTLC on the BTC payment channel. Once it arrives he releases `r_preimage`. This allows **both** the taker and the maker payments to finalize.

> 9. Both nodes locally mark the swap as completed.

Reviewer: ```suggestion
9. Both nodes locally mark the swap as completed once the respective HTLC was resolved and payment finalized.
```

> 9. Both nodes locally mark the swap as completed.

Reviewer: ```suggestion
8. Both nodes locally mark the swap as completed once the respective HTLC was resolved and payment finalized.
```
